getline.ts/src/blockchain.ts

```diff
@@ -263,21 +279,34 @@ export class GetlineBlockchain {
         const contract = this.web3.eth.contract(abi);
         const bytecode = await this.metabackend.getBytecode(contractName);
 
+        const gas: number = await (new Promise<number>((resolve, reject) => {
+            const fallback = 2000000;
+            const data = contract.getData(...params, { data: bytecode });
```

Comment:
I'm not sure this is the correct one. use `contract.new.getData()` just to be safe?

Reply:
As far as I can tell they're the same thing.

https://github.com/ethereum/web3.js/blob/develop/lib/web3/contract.js#L237

Reply:
https://github.com/ethereum/web3.js/commit/29e3efb8d88d41e1a7ef0fdfba0d652b503d63e0